packages/lesmis/src/components/MiniMap.js

```diff
@@ -24,14 +25,38 @@ const Map = styled(MapContainer)`
   height: auto;
 `;
 
+const BasicPolygon = styled(Polygon)`
+  fill: ${RED};
```

Comment:
```suggestion
fill: ${props => props.theme.palette.primary.main};
```

This is totally an optional change. Just thought I'd point out that it is possible to use the main theme color.